protos/escrow/escrow.proto

```diff
@@ -61,6 +62,12 @@ enum Schedule {
   CUSTOMIZED = 3;
 }
 
+enum ContractType {
+  REGULAR = 0;
+  PLAN = 1;
+  REVISE = 2;
```

Comment:
Regular status for original contract and Plan for renew contract, Revise status may not necessary for the change?

Reply:
Just add should be fine, all current contract should be regular.